lightning/src/ln/shutdown_tests.rs

```diff
@@ -71,6 +71,9 @@ fn pre_funding_lock_shutdown_test() {
 	check_closed_event!(nodes[1], 1, ClosureReason::CooperativeClosure);
 }
 
+// TODO: Determine why get_route fails when replacing get_payment_preimage_hash with
```

Comment:
You have another todo here.

Reply:
Mainly to note why it may be that a simple replacement doesn't work here. Can drop it if you prefer.